captum/robust/_core/attack.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+from typing import Callable
+
+from captum.robust.perturbation import Perturbation
+
+
+class Attack(Perturbation):
```

Comment:
Perturbation and Attack look so similar. Maybe we can just have Attack (it already has perturb method). Is `Perturbation` class necessary ?

Reply:
Yes, this is a good point. I originally only had Attack, but since we were generalizing to random perturbations which were independent of the model, I added the base Perturbation class to address the review comments and made it to not require the model in the constructor (while attack implies a white or black box attack which takes the model forward). I think to keep the naming generic, we could just keep the Perturbation base class and remove the Attack class, what do you think?

Reply:
That sounds good to me too. That will make things simpler.